tests/test_operator.py

```diff
@@ -30,6 +30,22 @@ def symbol(name, dimensions, value=0., mode='function'):
     return s.indexify() if mode == 'indexed' else s
 
 
+class TestAPI(object):
+
+    @classmethod
+    def setup_class(cls):
+        clear_cache()
+
+    def test_code_generation(self):
```

Comment:
This test, called **_code_generation**, was meant to verify the presence of the constants in the generated code (e.g. by searching for strings or inspecting ASTs through tools.pprint). Could you add such a test? Probably it can be generalised to non-constant data objects

The current test implementation, instead, compares computed values, which is also useful, but probably should be called differently (besides being added a docstring)